x-pack/plugins/maps/public/connected_components/add_layer_panel/flyout_body/layer_wizard_select.tsx

```diff
@@ -51,27 +135,40 @@ export class LayerWizardSelect extends Component<Props, State> {
         </div>
       );
     }
-    return this.state.layerWizards.map((layerWizard: LayerWizard) => {
-      const icon = layerWizard.icon ? <EuiIcon type={layerWizard.icon} size="l" /> : undefined;
 
-      const onClick = () => {
-        this.props.onSelect(layerWizard);
-      };
+    const wizardCards = this.state.layerWizards
+      .filter((layerWizard: LayerWizard) => {
+        return this.state.selectedCategory
+          ? layerWizard.categories.includes(this.state.selectedCategory!)
+          : true;
+      })
+      .map((layerWizard: LayerWizard) => {
+        const icon = layerWizard.icon ? <EuiIcon type={layerWizard.icon} size="l" /> : undefined;
 
-      return (
-        <Fragment key={layerWizard.title}>
-          <EuiSpacer size="s" />
-          <EuiCard
-            className="mapLayerAddpanel__card"
-            title={layerWizard.title}
-            icon={icon}
-            onClick={onClick}
-            description={layerWizard.description}
-            layout="horizontal"
-            data-test-subj={_.camelCase(layerWizard.title)}
-          />
-        </Fragment>
-      );
-    });
+        const onClick = () => {
+          this.props.onSelect(layerWizard);
+        };
+
+        return (
+          <EuiFlexItem key={layerWizard.title}>
+            <EuiCard
+              title={layerWizard.title}
+              icon={icon}
+              onClick={onClick}
+              description={layerWizard.description}
+              data-test-subj={_.camelCase(layerWizard.title)}
+            />
+          </EuiFlexItem>
+        );
+      });
+
+    return (
+      <>
+        {this._renderCategoryFacets()}
+        <EuiFlexGrid columns={2} gutterSize="s">
+          {wizardCards}
+        </EuiFlexGrid>
```

Comment:
I think it looks better to have a small space bellow the category facets. And also have the `EuiFlexGrid` with a medium gutter size. Same size as the layer panel margins.
```suggestion
        {this._renderCategoryFacets()}
        <EuiSpacer size="xs" />
        <EuiFlexGrid columns={2} gutterSize="m">
          {wizardCards}
        </EuiFlexGrid>
```